docs/topics/auth/default.txt

```diff
@@ -1691,7 +1691,25 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
 
 .. class:: UserCreationForm
 
-    A form for creating a new user.
+    A form for creating a new user
+
+    **Attributes:**
+
+    * ``username``:
+        The username sent to the form.
+
+    * ``password1``:
+        The password sent to the form.
+
+    * ``password2``:
+        The password confirmation sent to the form.
+
+    Because ``UserCreationForm`` extends ``django.forms.ModelForm`` it
```

Comment:
:confused: I don't know what you mean here. AFAIK `user`, `password1` and `password2` are all unique to this form